CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+# 84.0.0
+
+In this release we corrected a text field accesssibility behavior added ripple to chips and added
+additional delegate calls to post bottom sheet's scrolling.
+
+We finished updating additional theming docs for multiple components including chips, dialogs, cards.
+
+## Breaking changes
+
+### Text Field
+
+We fixed the accessibilty behavior of MDCTextField. We were using the value instead of the label.
+There is now a `,` that results in a pause between the label and the helper text. More information
+can be found [in the PR](https://github.com/material-components/material-components-ios/pull/7256)
```

Comment:
Please explain why this is a breaking change and what a client can do to address the breakage (i.e. link to a migration guide or include inline steps here).

Reply:
Added migration guide code snippet inline.